doc/telestrat.sgml

```diff
@@ -109,6 +150,22 @@ structures; accessing the struct fields will access the chip registers.
 
 </descrip><p>
 
+<descrip>
+
+  <tag><tt/VIA2/</tag>
+  Access to the VIA2 (Versatile Interface Adapter) chip is available via the
+  <tt/VIA2/ variable. The structure behind this variable is explained in <tt/_6522.h/.
+
+</descrip><p>
+
+<descrip>
+
+  <tag><tt/6551/</tag>
```

Comment:
That is a number, not a symbol name.  I suggest that you use `ACIA` as the chip name.

Reply:
Fixed